django_cloud_deploy/integration_tests/tool_requirements_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for tool_requirements.py."""
+import os
+
+from django_cloud_deploy import tool_requirements
+from django_cloud_deploy.cli import io
+from absl.testing import absltest
+
+
+class ToolRequirementsNegativeTest(absltest.TestCase):
+    """This will test all negative cases (not having the requirement).
+       It will remove any requirements in the setup.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Modify the path to exclude the necesary requirments."""
+        os.environ['PATH'] = ''
```

Comment:
Will this affect other test cases?

Reply:
I dont know, going to fix it just in case

Reply:
Was hoping to make these tests integration tests. If we mock that out wouldn't it decrease the value of the tests?

Reply:
currently by modifying the path, it seems to display integration level behavior. It's not slow, and with the setup/cleanup we can guarantee to not meddle with other tests (as well as they don't get run in parralell)